.github/workflows/deployment.yml

```diff
@@ -44,6 +36,7 @@ jobs:
 
   Chrome:
     needs: Build
+    if: github.event_name == 'push' || needs.Build.outputs.created
```

Comment:
Question: do we have to check if `created == 'yes'`, or just checking for the existence of that key would be fine?